DeepMineMod/DeepMineMod.csproj

```diff
@@ -28,6 +28,9 @@
     <Reference Include="UnityEngine.ImageConversionModule">
       <HintPath>Libs\UnityEngine.ImageConversionModule.dll</HintPath>
     </Reference>
+    <Reference Include="com.unity.multiplayer-hlapi.Runtime">
```

Comment:
Any reason for including the multiplayer-hlapi.Runtime? Not opposed to it if it adds some convenience/functionality.

Reply:
Yes, unfortunately `Thing` is a NetworkObject which lives in the multiplayer DLL, so patching `Thing. LoadPrefabs` using `typeof(Thing)` requires it to be referenced in the protect AFAICT.

Reply:
It would be nice if there's a way to rewrite the HarmonyPatch to not reference Thing or if we can get away with not including the library, but I couldn't convince my VS to build the lib without referencing it.

Reply:
Okay! Thanks for clarifying.